frontend/src/app/admin/premium-domains/page.tsx

```
@@ -0,0 +1,1204 @@
+                                                                                            </button>
+                                                                                        </div>
+                                                                                    </div>
+                                                                                ) : (
+                                                                                    <p
+                                                                                        className="text-gray-700 dark:text-gray-300 text-sm cursor-pointer hover:text-yellow-600 dark:hover:text-yellow-400"
+                                                                                        onClick={() => startEditingQuestion(question)}
+                                                                                        dangerouslySetInnerHTML={{
+                                                                                            __html: safeRenderHTML(question.question_text),
+                                                                                        }}
+                                                                                    />
+                                                                                )}
```

Biome (2.1.2)

[error] Enforce to have the onClick mouse event with the onKeyUp, the onKeyDown, or the onKeyPress keyboard event.

Actions triggered using mouse events should have corresponding keyboard events to account for keyboard-only navigation.

(lint/a11y/useKeyWithClickEvents)